src/BlockTypes/ProductCollection.php

```diff
@@ -139,14 +144,17 @@ public function build_query( $query, $block, $page ) {
 			's'              => $block_context_query['search'],
 		);
 
-		$is_on_sale = $block_context_query['woocommerceOnSale'] ?? false;
+		$is_on_sale       = $block_context_query['woocommerceOnSale'] ?? false;
+		$taxonomies_query = $this->get_filter_by_taxonomies_query( $query['tax_query'] );
```

Comment:
I'm getting this PHP warning:

```
Warning: Undefined array key "tax_query" in /wp-content/plugins/woocommerce-blocks/src/BlockTypes/ProductCollection.php` on line 148
```

Reply:
I see them directly on the screen. I think you need to set these values in `wp-config.php`:

```PHP
define( 'WP_DEBUG', true );
define( 'SCRIPT_DEBUG', true );
```